R/format_ae_forestly.R

```diff
@@ -224,6 +225,11 @@ format_ae_forestly <- function(
     name = reactable::colDef(
       header = "Adverse Events",
       minWidth = width_term, align = "right"
+    ),
+    soc_name = reactable::colDef(
+      header = "SOC Name",
+      minWidth = width_term, align = "right",
+      show = F
```

Comment:
Prefer `TRUE` and `FALSE` over `T` and `F`. https://style.tidyverse.org/syntax.html#logical-vectors